settings/authentication-personalization/authentication-vs-personalization.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Authentication vs Personalization'
+description: 'How to determine which product is best for you'
+---
+
+Mintlify offers both Authentication and Personalization. For the most part, Authentication is
+just Personalization + privacy. However, there are some key differences that are important to
+be aware of, from most important to least:
+
+### Security Guarantees
+
+Authentication is a fully-fledged private documentation solution. Every aspect of your docs,
+including page content, images, search results, and AI chat features, are completely inaccessible to
+unauthenticated users.
+
+Personalization, on the other hand, offers no security guarantees for your documentation content.
+All page content, images, search results, and AI chat features can be accessed by the public. Even
+if you are using the hidden pages feature of Personalization, a motivated attacker would still
+be able to access the content of a hidden page.
+
+### Handshake Methods
+
+Due to the difference in security requirements for Authentication and Personalization, different
+handshake methods are available for each. Both methods offer a JWT and OAuth Handshake, although
+the setup steps are slightly different.
+
+Authentication offers two additional Handshake methods:
+
+- **Password Handshake**, which protects your website with a single configurable global password.
+
+- **Mintlify Dashboard Handshake**, which will allow users to view your documentation only if they have
+access to your dashboard.
+
+Personalization offers one additional Handshake method:
+
+- **Shared Session Handshake**, a super simple method which only requires setting up a single endpoint
+that returns data for already-authenticated users.
+
+{/* 
+### Pricing
+
+Mintlify strives to ensure that your private content stays private, full stop. Because this level
+of security is not easy to achieve, Authentication is an enterprise feature. If an enterprise plan
+doesn't make sense for you, consider whether Personalization might meet your needs. Personalization
+is available on all plans.
+*/}
```

Comment:
loll can we completely delete this

Reply:
(instead of commenting it out)

Reply:
Yup, although I think it would be a valuable thing to add back if we ever decide to put user auth on a different plan